packetbeat/pb/event.go

```diff
@@ -336,13 +340,33 @@ func marshalStruct(m common.MapStr, key string, val reflect.Value) error {
 			break
```

Comment:
I know you didn't change this code, but, am I wrong in thinking this should be a `continue` instead of a `break`?
If so, can you add a commit fixing this please?

Reply:
wow, good eye, totally didn't catch that--i'm pretty sure you're right seeing as this would terminate serialization if you hit an untagged field in the struct.